---
title: "Publications"
layout: default
excerpt: "RT2 Lab -- Publications."
sitemap: false
permalink: /publications/
---
<h1>{{page.title}}</h1>
<div class="panel-group" id="accordion">
   <div class="panel panel-default" >
      <h3 style="margin-left: 5px">Books</h3>
   </div>
   <div class="panel-heading">1. KC, DB (editor) Methods in Molecular Biology, Computational Prediction of Post-translational Modification Sites, 2022, Springer US.
   </div>
   <div class="panel panel-default" >
      <h3 style="margin-left: 5px">Book Chapter in Press</h3>
   </div>
   <div class="panel-heading">1. Pokharel S*, Sidorov E*, Caragea D, KC DB#, NLP-based encoding techniques for prediction of post-translational modification sites and protein functions, in Machine Learning in Bioinformatics of Protein Sequences, Editor: Kurgan, L, 2023 (In press).
   </div>
   <div class="panel panel-default" >
      <h3 style="margin-left: 5px">Book Chapters</h3>
   </div>
   <div class="panel-heading">1. Al-Barakati H*, Newman RH, KC DB#, Poole LB#, Bioinformatic Analyses of Peroxiredoxins and RF-Prx: A random forest-based predictor and classifier for Prxs, Methods in Molecular Biology: Computational Methods for predicting post-translational modification sites, Editor, KC DB, 2022, 155-176, 2022.
   </div>
   <div class="panel-heading">2. Pakhrin SC*, Pokharel S*, Saigo H, KC DB#, Deep Learning–Based Advances In Protein Post-translational Modification Site and Protein Cleavage Prediction, Methods in Molecular Biology: Computational Methods for predicting post-translational modification sites, Editor, KC DB, 285-322, 2022.
   </div>
   <div class="panel-heading">3. Ismail H*, White C*, Al-Barakati H*, Newman RH, KC DB#, FEPS: A tool for feature extraction from protein sequence, Methods in Molecular Biology: Computational Methods for predicting post-translational modification sites, Editor, KC DB, 285-322, 2022
   </div>
   <div class="panel-heading">7. MacCarthy E*, Perry D*, KC DB#, Advances in proteins super-secondary structure prediction and application to protein structure prediction, Methods in Molecular Biology: Protein Supersecondary structure, Editor: Kister A, 2019; 1958:15-4.
   </div>
   <div class="panel-heading">8. Dennis R Livesay, KC DB, David, La, Predicting protein functional sites with phylogenetic motifs: Past, present and beyond. To appear in Omics approaches for protein function prediction, Kihara D (Ed.), Springer, 2010.
   </div>
   <div class="panel-heading">9. KC DB, Dennis R Livesay, A spectrum of phylogenetic-based approaches for predicting protein functional sites, In Bioinformatics for Systems Biology, Krawetz S (Ed.), Humana Press. ISBN: 978-1-934115-02-2, 2009.
   </div>
   <div class="panel panel-default" >
      <h3 style="margin-left: 5px">Journal Papers Submitted</h3>
   </div>
   <div class="panel-heading">1. Pratyush P*, Pokharel S*, Hiroto S, KC DB#, pLMSNOSite:  prediction of protein s-nitrosylation sites using embeddings from language model, BMC Bioinformatics, submitted.
   </div>
   <div class="panel-heading">2. Pakhrin S*, Pokharel S*, Aoki-kinoshita KF, Caragea D, KC DB#, LMNGlyPred: prediction of human n-lined glycosylation site using embeddings from pre-trained protein language model, Glycobioly, Submitted. 
   </div>
   
   <div class="panel panel-default" >
      <h3 style="margin-left: 5px">Journal Papers - Accepted</h3>
   </div>
   <div class="panel-heading">1. Pokharel S*, Pratyush P*, Heinzinger M, Newman RH, KC DB#, LMSuccSite: Improving Protein succinylation sites prediction using embeddings from protein language model, Scientific Reports, In Press.
   </div>
   <div class="panel panel-default" >
      <h3 style="margin-left: 5px">Journal Papers – Published </h3>
   </div>
   <div class="panel-heading">1. Maccarthy E*, Zhang C, Zhang Y, KC DB#, GPU-ITASSER: a GPU accelerated I-TASSER protein structure prediction tool, Bioinformatics, 38(6):1754-1755, 2022.
   </div>
   <div class="panel-heading">2.  Saigo H, KC DB, Saito N, Einstein-Roscoe regression for the slag viscosity problem in steelmaking, Scientific Reports, 12(1), 1-9, 2022. 
   </div>
   <div class="panel-heading">3.  Pakhrin SC*, Aoki-Kinoshita KF, Caragea D, KC DB#, DeepNGlyPred: A Deep neural network-based approach for human N-linked glycosylation site prediction, Molecules, 27(23), 7314, 2021. </div>
   <div class="panel-heading">4.  Chaudhari M*, Thapa N*, Ismail H*, Chopade S*, Caragea D, Kohn M, Newman RH, KC DB#, DTL-DephosSite: Deep transfer learning-based approach to predict dephosphorylation sites, Front. Cell Dev. Biol., 662983, 2021.</div>
   <div class="panel-heading">5.  Pakhrin SC*, Shrestha B, Adhikari B, KC DB#, Deep Learning-based advances in protein structure prediction, Int. J. Mol. Sci, 2032, 22(11), 5553, 2021.</div>
   <div class="panel-heading">6.  Thapa N*, Chaudhari M*, Iannetta A, White C, Roy K, Newman RH, Hicks LM, KC DB#, A deep learning based approach for prediction of Chlamydomonas reinhardtii phosphorylation sites, Scientific Reports, 11, 12550, 2021.</div>
   <div class="panel-heading">7.  Thapa N*, Liu, Z, KC DB,  Gokaraju B,  Roy K, Comparison of machine learning and deep learning models for network intrusion detection systems, Future Internet, Vol. 12, 10, 167, 2020.</div>
   <div class="panel-heading">8.  Chaudhari M*, Thapa N*, Roy K, Newan RH, Saigo H, KC DB#, DeepRMethylSite: A Deep Learning Based Approach for prediction of arginine methylation sites in Proteins, Molecular Omics, 16, 448-454, 2020.</div>
   <div class="panel-heading">9.  Thapa N*, Chaudhari M*, McManus S, Roy, K, Newman RH, Saigo H, KC DB#  
      DeepSuccinylSite: a deep learning based approach for protein succinylation site prediction, BMC Bioinformatics, 21(3), 1-10, 2020. 
   </div>
   <div class="panel-heading">10. Al-barakati H*, Thapa N*, Saigo H, Roy K, Newman RH, KC DB#, RF-MaloSite and DL-MaloSite: Methods based on random forest and deep learning to identify malonylation sites, Computational Structural Biotechnology Journal, 18:852-860, 2020.</div>
   <div class="panel-heading">11. Albarakati H*, Saigo H, Newman RH, KC DB#, RF-GlutarySite: Random Forest based predictor for Glutarylation sites, Molecular Omics, doi, 10.1039, 2019. (Selected for cover article)</div>
   <div class="panel-heading">12. Ismail H*, Saigo H, KC DB#, RF-NR: Random forest based approach for improved classification of nuclear receptors, In IEEE ACM Transactions on Bioinformatics and Computational Biology, Vol 15, No 6., 2018. </div>
   <div class="panel-heading">13. Albarakati H*, McConnell EW, Hicks, LM, Poole LB, Newman RH, KC DB#, SVM-SulfoSite: A support vector machine based predictor for sulfenylation sites, Scientific Reports, 8: 11288, 2018.</div>
   <div class="panel-heading">14. Amini H, Wang LJ, Hasemisohi A, Bikdash M, KC DB, Yuan WQ, An integrated growth kinetics and computational fluid dynamics model for the analysis of algal productivity in open raceway ponds, Computers and Electronics in Agriculture, 145, 363-372, 2018.</div>
   <div class="panel-heading">15. White C*, Ismail H*, Saigo H., KC DB#, CNN-BLPred: A Convolutional Neural Network based predictor for Beta lactamases (BL) and their class, BMC Bioinformatics, 18(Suppl 16):577, 2017.</div>
   <div class="panel-heading">16. KC DB#, Recent advances in sequence-based protein structure prediction 
      Briefings in Bioinformatics,1:18(6):1021-1032, 2017.
   </div>
   <div class="panel-heading">17. Chapman CH*, Adami C, Wilke CO, KC DB# 
      The evolution of logic circuits for the purpose of protein contact map prediction 
      PeerJ, 5:e3139 https://doi.org/10.7717/peerj.3139, 2017.
   </div>
   <div class="panel-heading">18. Ismail H*, Newman RH, KC DB#, RF-Hydroxysite: a random forest based predictor for hydroxylation sites 
      Molecular Biosystems, 12(8):2427-35, 2016
   </div>
   <div class="panel-heading">19. Jha A*, Flurchick KM, Bidash M, KC DB# 
      Parallel-SymD: A parallel approach to detect internal symmetry in protein domains
      BioMed Research International, 2016, 4628592.
   </div>
   <div class="panel-heading">20. Jiang Y,…, Chapman CH*, KC DB, …., Predrag Radivojac, An expanded evaluation of protein function prediction methods show an improvement in accuracy, Genome Biology, 17:184, 2016.</div>
   <div class="panel-heading">21. Ismail H*, Jones A*, Kim J., Newman R., KC DB#: Improved Random Forest based prediction of phosphorylation sites, Biomed Research International, 2016, 3281590.</div>
   <div class="panel-heading">22. Amini H, Shahbazi A, Bikdash M, KC DB, Yuan W, Hashemisohi A, Wang L 
      Numerical and experimental investigation of hydrodynamics and light transfer in open raceway ponds at various algal cell concentrations and medium depths 
      Chemical Engineering Science 156, 11-23, 2016.
   </div>
   <div class="panel-heading">23.  Khan IK, Wei Q, Chapman S*, KC DB, Kihara D: PFP and ESG protein function prediction methods in 2014: Effects of database updates and ensemble approaches, Giga Science, 4:43, 2015.</div>
   <div class="panel-heading">24.  CH Tai, Rohit P, KC DB, Shilling J, Lee BK, SymD webserver: a platform for detecting internally symmetric protein structures, Nucleic Acids Res, 42(W1):W296-W300, 2014. </div>
   <div class="panel-heading">25. Yu Y, Megri AC, Flurchick KM, KC DB, The improvement of the computational performance of the zonal model POMA using parallel techniques, American Journal of Engineering and Applied Sciences 7(1): 185-193, 2014, Science Publication.</div>
   <div class="panel-heading">26.  KC DB#, Structure-based methods for computational protein functional site prediction,  Comput Struct Biotechnology J, 8: e201308005, 2013.</div>
   <div class="panel-heading">27. KC DB and Livesay DR, Topology improves phylogenetic motif functional site predictions, IEEE/ACM Transactions on Computational Biology and Bioinformatics, 8:226-233, 2011.</div>
   <div class="panel-heading">28.  KC DB#, Improving Consensus Structure by Eliminating Averaging Artifacts, BMC Structural Biology, 9:12, 2009.</div>
   <div class="panel-heading">29.  KC DB and Livesay DR, Improving position specific predictions of protein functional sites using phylogenetic motifs, Bioinformatics, 24:2308-2316, 2008.</div>
   <div class="panel-heading">30.  Brown JB, KC DB, Tomita E, Akutsu T, Multiple methods for protein side chain packing using maximum weight cliques, Genome Informatics, 17-1, 3-12, 2006.</div>
   <div class="panel-heading">31.  Akutsu T, Hayashida M, KC DB, Tomita E, Suzuki J,  Horimoto K, Protein Threading by Dynamic Programming and Clique Based Approaches, IEICE Transactions on Fundamentals of Electronics, Communication and Computer Science, E89-A, 1215-1222, 2006.</div>
   <div class="panel-heading">32.  KC DB, Jun’ichi Suzuki J, Horimoto K, Akutsu T, Protein Threading With Profiles and Distance Constraints Using Clique Based Algorithms, Journal of Bioinformatics and Computational Biology, 4:19-42, 2006.</div>
   <div class="panel-heading">33.  Moesa HA, KC DB, Akutsu T, Efficient Determination of Cluster Boundaries for Analysis of Gene Expression Profile Data Using Hierarchical Clustering and Wavelet Transform, Genome Informatics, 16(1): 132-141, 2005.</div>
   <div class="panel-heading">34.  KC DB, Tomita E, Suzuki J, Akutsu T, Protein side-chain packing problem: A maximum common edge-weight clique algorithmic approach, Journal of Bioinformatics and Computational Biology, 3(1): 103-126, 2005.</div>
   <div class="panel-heading">35. KC DB, Akutsu T, Tomita E, Seki T, Fujiyama A, Point Matching Under Non-uniform Distortions and Protein Side Chain Packing Based on an Efficient Clique Algorithm, Genome Informatics, 13: 143-152, 2002.
   </div>
</div>
